packages/loading/src/directive.ts

```diff
@@ -1,34 +1,37 @@
 import Loading from './index'
 
+const createInstance = (el, binding) => {
+  const textExr = el.getAttribute('element-loading-text')
+  const spinnerExr = el.getAttribute('element-loading-spinner')
+  const backgroundExr = el.getAttribute('element-loading-background')
+  const customClassExr = el.getAttribute('element-loading-custom-class')
+  const vm = binding.instance
+  el.instance = Loading({
+    text: vm && vm[textExr] || textExr,
+    spinner: vm && vm[spinnerExr] || spinnerExr,
+    background: vm && vm[backgroundExr] || backgroundExr,
+    customClass: vm && vm[customClassExr] || customClassExr,
+    fullscreen: !!binding.modifiers.fullscreen,
+    target: !!binding.modifiers.fullscreen ? null : el,
+    body: !!binding.modifiers.body,
+    visible: !!binding.value,
+    lock: !!binding.modifiers.lock,
+  })
+}
+
 const vLoading = {
   mounted(el, binding) {
-    const textExr = el.getAttribute('element-loading-text')
-    const spinnerExr = el.getAttribute('element-loading-spinner')
-    const backgroundExr = el.getAttribute('element-loading-background')
-    const customClassExr = el.getAttribute('element-loading-custom-class')
-    const vm = binding.instance
-    const instance = Loading({
-      text: vm && vm[textExr] || textExr,
-      spinner: vm && vm[spinnerExr] || spinnerExr,
-      background: vm && vm[backgroundExr] || backgroundExr,
-      customClass: vm && vm[customClassExr] || customClassExr,
-      fullscreen: !!binding.modifiers.fullscreen,
-      target: !!binding.modifiers.fullscreen ? null : el,
-      body: !!binding.modifiers.body,
-      visible: !!binding.value,
-      lock: !!binding.modifiers.lock,
-    })
-    el.instance = instance
+    if(!!binding.value){
```

Comment:
Since it's bool type, why casting it anyway?

Reply:
These code was wrote by @bastarder. Maybe because the `binding.value` may be not a boolean typle value, there seems to be no way to enforce restrictions on directive to recept a boolean type value.

Reply:
Got it. We should constraint the type then. Let's keep it this way.